### PR TITLE
update document how to use multiple extensions

### PR DESCRIPTION
**Pull request by hanapei00**
**Replace this text with a summary of the changes in your PR. The more detailed you are, the better.**

there is not written how to use multiple extensions. in the other wise, rubocop-performance and rubocop-rails are written it.

ref:
 https://github.com/rubocop-hq/rubocop-rails#rubocop-configuration-file
 https://github.com/rubocop-hq/rubocop-performance#rubocop-configuration-file

So this pull request update document :)

---

Before submitting the PR make sure the following are checked:

* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [changelog](https://github.com/rubocop-hq/rubocop-rspec/blob/master/CHANGELOG.md) if the new code introduces user-observable changes.
* [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

If you have created a new cop:

* [x] Added the new cop to `config/default.yml`.
* [x] The cop documents examples of good and bad code.
* [x] The tests assert both that bad code is reported and that good code is not reported.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,14 @@ cops together with the standard cops.
 rubocop --require rubocop-rspec
 ```
 
+Alternatively, use the following array notation when specifying multiple extensions.
```
Comment (pirj) [inline README.md]: I believe this should go to the previous section, after line 38.

File: README.md [hunk 2]
```diff
@@ -45,6 +45,14 @@ cops together with the standard cops.
...
+
+```bash
```
Comment (pirj) [inline README.md]: This is YAML, so in order to make it consistent with the other guides, you may consider writing this as:
```
```yaml
```

Comment (pirj) [reply README.md]: Appreciate if you could add `yaml` on the line 35.

File: README.md (hunk 1 above)
Comment (hanapei00) [reply README.md]: thank you, surely after line 38 is good

File: README.md (hunk 2 above)
Comment (hanapei00) [reply README.md]: ok :)

Comment (hanapei00) [reply README.md]: fixed it!

https://github.com/rubocop-hq/rubocop-rspec/pull/783/commits/1eede967354d99c0ffaf1ca57d7fbfc0c9a1d8cc

File: README.md (hunk 1 above)
Comment (hanapei00) [reply README.md]: fixed it!

https://github.com/rubocop-hq/rubocop-rspec/pull/783/commits/1eede967354d99c0ffaf1ca57d7fbfc0c9a1d8cc

**Pull request merged**